# Board configuration for the environment


boards:
  #Possible dictionaries:
  # DICT_4X4_50: 0
  # DICT_4X4_100: 1
  # DICT_4X4_250: 2
  # DICT_4X4_1000: 3
  # DICT_5X5_50: 4
  # DICT_5X5_100: 5
  # DICT_5X5_250: 6
  # DICT_5X5_1000: 7
  # DICT_6X6_50: 8
  # DICT_6X6_100: 9
  # DICT_6X6_250: 10
  # DICT_6X6_1000: 11
  # DICT_7X7_50: 12
  # DICT_7X7_100: 13
  # DICT_7X7_250: 14
  # DICT_7X7_1000: 15
  # DICT_ARUCO_ORIGINAL: 16
  dictionary_id: 0

  #Sizes should be set to some factor of the output
  # resolution when generating images
  #Sizes should reflect the real life marker sizes
  # when detecting
  marker_size: 0.05
  marker_spacing: 0.01

  border_bits: 1

  output_height: 800
  output_width: 800

  num_boards: 4

  board_0:
    rows: 2
    cols: 2
    ids: [31, 32, 33, 34]

  board_1:
    rows: 2
    cols: 3
    ids: [25,26,27,28,29,30]

  board_2:
    rows: 1
    cols: 1
    ids: [49]

  board_3:
    rows: 4
    cols: 4
    ids: []
